File: problem1.js
function sumMultiplesOfNum (num) {
  var result = []
  for (let i = 2; i < num; i++) {
    if ((i % 3 == 0) || (i % 5 == 0)) {
      result.push(i)
    }
  }
  return result.reduce(function (sum, value) {
    return sum + value
  })
}

File: problem1.rb
def sum_multiples_of_num(num)
  result = []
  2.upto(num-1) do |i|

    if i % 3 == 0 || i  % 5 == 0
      result << i
    end
  end
  result.inject(0){|sum,x| sum + x}
end